ide/che-core-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionEditorPresenter.java

```diff
@@ -1210,7 +1211,15 @@ public void onContextMenu(ContextMenuEvent event) {
                       new OrionVcsChangeMarkersRuler(
                           orionExtRulerOverlay, editorWidget.getEditor());
 
-                  String vcsName = appContext.getRootProject().getAttribute("vcs.provider.name");
+                  VirtualFile file = input.getFile();
+                  Project project =
+                      file instanceof Resource ? ((Resource) file).getProject() : null;
+                  if (project == null) {
+                    resolve.apply(null);
+                    return;
+                  }
+
+                  String vcsName = project.getAttribute("vcs.provider.name");
```

Comment:
Can you apply checking on the null? User can delete folder .git but attribute "vcs.provider.name" for project still exist...